cloudinit/distros/openbsd.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2019-2020 Gonéri Le Bouder
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import os
+import platform
+
+import cloudinit.distros.netbsd
+from cloudinit import log as logging
+from cloudinit import util
+
+LOG = logging.getLogger(__name__)
+
+
+class Distro(cloudinit.distros.netbsd.NetBSD):
```

Comment:
@OddBloke the class now inherite from the NetBSD class.

Reply:
Thanks, this looks a lot better!

Reply:
Great :-)